compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/ParagraphLayouter.skiko.kt

```diff
@@ -48,7 +55,7 @@ import org.jetbrains.skia.paragraph.Paragraph
  * reusedParagraph.layout(300f): 10.004400ms
  * builder.build().layout(300f): 23.421500ms
  */
-class ParagraphLayouter(
+internal class ParagraphLayouter(
     val text: String,
```

Comment:
```suggestion
class ParagraphLayouter(
    val text: String,
```

ParagraphLayouter already was public, and making it internal - it's not a backward compatible change.

Reply:
Also you can choose `@InternalComposeUiApi`

Reply:
@igordmn What do you think?
Maybe this class needs public access for tests?

Reply:
I think we can do so in this case, but the final decision is for @igordmn

Reply:
Decided to include this to our API policy:
```
If API is not documented, accidentaly made public, don't make sense as public API, and isn't used by anyone, we can mark it as internal straight away, but consult that with the author of the API.
```